lib/ansible/modules/network/cumulus/nclu.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016-2017, Cumulus Networks <ce-ceng@cumulusnetworks.com>
+#
+# This file is part of Ansible
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+DOCUMENTATION = '''
+---
+module: nclu
+version_added: "2.2.3"
+author: "Cumulus Networks"
+short_description: Configure network interfaces using NCLU
+description:
+    - Interface to the Network Command Line Utility, developed to make it easier
+      to configure operating systems running ifupdown2 and Quagga, such as
+      Cumulus Linux. Command documentation is available at
+      https://docs.cumulusnetworks.com/display/DOCS/Network+Command+Line+Utility
```

Comment:
To get a URL use the following:
`U(https://docs.cumulusnetworks.com/display/DOCS/Network+Command+Line+Utility)`

See http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html for details